docs/content/how-to/common/nodepool-version-compatibility.md

```
@@ -0,0 +1,38 @@
+
+## Version Compatibility Rules
+
+### Supporting Version Rules
+1. NodePool version cannot be higher than the HostedCluster version.
+2. For 4.y versions:
+    - For 4.even versions (e.g. 4.18), allows up to 2 minor version differences (4.17, 4.16);
+    - For 4.odd versions (e.g. 4.17), allows up to 1 minor version difference (4.16).
+
+### Examples
+- If HostedCluster version is 4.18.5:
+    - Allowed NodePool versions: 4.18.5, 4.18.4, 4.17.z, 4.16.z
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)